### PR TITLE
Sign in link campaign codes

### PR DESCRIPTION
**Pull request by NathanielBennett**
Adds the folllowing campaign codes to sign in links

Link                                             INTCAM
Save for later - article             DOTCOM_ARTICLE_SFL
Become a member' - header    DOTCOM_HEADER_BECOMEMEMBER
Sign in - header                     DOTCOM_HEADER_SIGNIN
Comments - sign in                   DOTCOM_COMMENTS_SIGNIN
Comments - register          DOTCOM_COMMENTS_REG
Email sign up                            DOTCOM_NETWORKFRONT_EMAIL ( comes from Fronts Tool so you can't see it here )

### CODE REVIEW

File: common/app/views/fragments/discussionFooter.scala.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
         <div class="container__meta">
             <h2 class="container__meta__title">comments <span class="discussion__comment-count js-discussion-comment-count"></span></h2>
 
-            <p class="container__meta__item discussion__meta discussion__meta--open-signed-out"><a class="u-underline" href="@Configuration.id.url/signin">Sign in</a>
-                or <a class="u-underline" href="@Configuration.id.url/register">create your Guardian account</a> to join the discussion.
+            <p class="container__meta__item discussion__meta discussion__meta--open-signed-out"><a class="u-underline" href="@Configuration.id.url/signin&INTCMP=DOTCOM_COMMENTS_SIGNIN">Sign in</a>
+                or <a class="u-underline" href="@Configuration.id.url/register&INTCMP=DOTCOM_COMMENTS_REG">create your Guardian account</a> to join the discussion.
```
Comment (gtrufitt) [inline common/app/views/fragments/discussionFooter.scala.html]: Should this be ``register?INTCMP=DOTCOM_COMMENTS_REG`` instead of the &?

**Comment by NathanielBennett** [comment]
@gtrufitt good spot. Thanks!

File: common/app/views/fragments/discussionFooter.scala.html [hunk 2]
```diff
@@ -8,8 +8,8 @@
         <div class="container__meta">
             <h2 class="container__meta__title">comments <span class="discussion__comment-count js-discussion-comment-count"></span></h2>
 
-            <p class="container__meta__item discussion__meta discussion__meta--open-signed-out"><a class="u-underline" href="@Configuration.id.url/signin">Sign in</a>
-                or <a class="u-underline" href="@Configuration.id.url/register">create your Guardian account</a> to join the discussion.
+            <p class="container__meta__item discussion__meta discussion__meta--open-signed-out"><a class="u-underline" href="@Configuration.id.url/signin&INTCMP=DOTCOM_COMMENTS_SIGNIN">Sign in</a>
```
Comment (jennysivapalan) [inline common/app/views/fragments/discussionFooter.scala.html]: The `&` should be `?` on signin link?

File: common/app/views/fragments/header.scala.html [hunk 3]
```diff
@@ -19,7 +19,7 @@
                         <div class="brand-bar__item brand-bar__item--profile
                                     brand-bar__item--has-control brand-bar__item--register
                                     popup-container has-popup js-profile-register">
-                            <a href="@Configuration.id.url/register?skipConfirmation=true&INTCMP=HEADER_GU_REG&returnUrl=@URLEncode("https://membership.theguardian.com/join/friend/enter-details")"
+                            <a href="@Configuration.id.url/register?skipConfirmation=true&DOTCOM_HEADER_BECOMEMEMBER&returnUrl=@URLEncode("https://membership.theguardian.com/join/friend/enter-details")"
```
Comment (desbo) [inline common/app/views/fragments/header.scala.html]: should it be `INTCMP=DOTCOM_HEADER_BECOMEMEMBER`?

File: common/app/views/fragments/discussionFooter.scala.html (hunk 2 above)
Comment (NathanielBennett) [inline common/app/views/fragments/discussionFooter.scala.html]: @jennysivapalan thanks! :(

**Comment by desbo** [comment]
:+1:

**Pull request merged**